wasm/src/graphics/render.rs

```diff
@@ -26,37 +26,40 @@ impl<T: GraphicsApi> Render<T> {
             .map_err(|e| ClientError::WebGlError(format!("WebGl2 error: {}", e)))?;
         let size = (unsafe { SynchronizationMemory::get() }).canvas_size;
         self.graphics.update_size(size.0, size.1);
-        self.graphics.start_frame(&[0.0, 0.0, 0.0])?;
+        self.graphics.start_frame()?;
         if self.draw_sprites()? {
             self.frame_nr += 1;
         }
         self.graphics.end_frame()
     }
 
     pub fn upload_texture(&mut self, id: u32) -> Result<(), ClientError> {
-        let guard = RESOURCE_TABLE.read();
-        let texture = guard.get(id as usize)?;
-        self.graphics.resize_texture_pool(id + 1)?;
-        self.graphics.upload_texture(texture, id)?;
+        log::info!("upload texture {} to gpu", id);
```

Comment:
should probably be a `log::debug!`

Reply:
Jup, but that comment is to be deleted anyway. It was more of an accident, that it got commited